examples/orbit_5k_pipeline.ipynb

```diff
@@ -4,18 +4,29 @@
    "cell_type": "markdown",
```

Comment:
<p>1)Please keep:</p><h4>Author: Raphael Reinauer</h4><p>to make it consistent with the other notebooks</p><p> </p><p>2)Make it clearer:</p><p>consists of subsets of [0, 1]^2, each containing 1000 points.</p><p> </p><p>3) The recursion formula is not correct.</p><p> </p><p>4) We generate S^ρ 1000 times per values of ρ. -&gt; We generate 1000 orbits for each value of rho.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-deep/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>